app/index.html

```diff
@@ -6,13 +6,25 @@
     <!-- openmrs favicon -->
     <link rel="shortcut icon" type="image/ico" href="../../images/openmrs-favicon.ico" />
     <link rel="icon" type="image/png" href="../../images/openmrs-favicon.png" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="libs/bootstrap/dist/css/bootstrap.css">
+    <script src="libs/jquery/dist/jquery.min.js" ></script>
```

Comment:
Sorry for the late reply Dkayiwa. The libs folder is managed using a frontend dependency manager called bower. It's not ideal to push external dependencies to git. When you clone this project and run npm install the libs folder automatically gets created and all the dependencies gets installed.